docs/tutorial.md

```diff
@@ -255,77 +286,58 @@ tasks/build-docker-image-from-git-source   7m
 
-```yaml
-apiVersion: tekton.dev/v1alpha1
-kind: TaskRun
-metadata:
-  creationTimestamp: 2018-12-11T18:14:29Z
-  generation: 1
-  name: build-docker-image-from-git-source-task-run
-  namespace: default
-  resourceVersion: "6733537"
-  selfLink: /apis/tekton.dev/v1alpha1/namespaces/default/taskruns/build-docker-image-from-git-source-task-run
-  uid: 99d297fd-fd70-11e8-9129-42010a8a0fdc
-spec:
-  generation: 1
-  inputs:
-    params:
-      - name: pathToDockerFile
-        value: Dockerfile
-      - name: pathToContext
-        value: /workspace/git-source/examples/microservices/leeroy-web #configure: may change depending on your source
-    resources:
-      - name: git-source
-        paths: null
-        resourceRef:
-          name: skaffold-git
-  outputs:
-    resources:
-      - name: builtImage
-        paths: null
-        resourceRef:
-          name: skaffold-image-leeroy-web
-  taskRef:
-    name: build-docker-image-from-git-source
-  taskSpec: null
-status:
-  conditions:
-    - lastTransitionTime: 2018-12-11T18:15:09Z
-      status: "True"
-      type: Succeeded
-  podName: build-docker-image-from-git-source-task-run-pod-24d414
-  startTime: 2018-12-11T18:14:29Z
-  steps:
-    - terminated:
-        containerID: docker://138ce30c722eed....c830c9d9005a0542
-        exitCode: 0
-        finishedAt: 2018-12-11T18:14:47Z
-        reason: Completed
-        startedAt: 2018-12-11T18:14:47Z
-    - terminated:
-        containerID: docker://4a75136c029fb1....4c94b348d4f67744
-        exitCode: 0
-        finishedAt: 2018-12-11T18:14:48Z
-        reason: Completed
-        startedAt: 2018-12-11T18:14:48Z
-```
-
-The status of type `Succeeded = True` shows the Task ran successfully and you
+```
+Name:        build-docker-image-from-git-source-task-run
+Namespace:   default
+Task Ref:    build-docker-image-from-git-source
+
+Status
+STARTED       DURATION     STATUS
+2 hours ago   56 seconds   Succeeded
+
+Input Resources
+NAME            RESOURCE REF
+docker-source   skaffold-git
+
+Output Resources
+NAME         RESOURCE REF
+builtImage   skaffold-image-leeroy-web
+
+Params
+NAME               VALUE
+pathToDockerFile   Dockerfile
+pathToContext      /workspace/docker-source/examples/microservices/leeroy-web
+
+Steps
+NAME
+build-and-push
+create-dir-builtimage-wtjh9
+git-source-skaffold-git-tck6k
+image-digest-exporter-hlbsq
+```
+
+The status of type `Succeeded` shows the Task ran successfully and you
 can also validate the Docker image is created in the location specified in the
 resource definition.
 
+If you run into issues, use the following command to receive the logs:
+
+```bash
+tkn taskrun logs build-docker-image-from-git-source-task-run
+```
+
 ## Pipeline
 
-A [`Pipeline`](pipelines.md) defines a list of tasks to execute in order, while
-also indicating if any outputs should be used as inputs of a following task by
+A [`Pipeline`](pipelines.md) defines a list of `Tasks to execute in order, while
```

Comment:
Looks like you might have missed a backtick after Tasks here.

Reply:
good catch